backend/src/middleware/errorHandler.js

```diff
@@ -1,5 +1,5 @@
 const errorHandler = (err, req, res, _next) => {
-  console.error(err)
+  // console.error(err)
```

Comment:
i'd keep this. and also print the traceback - https://stackoverflow.com/questions/36645398/how-to-track-an-error-in-node-js-traceback

Reply:
ya shit i just removed that accidentally